app/presenters/print/alert_presenter.rb

```diff
@@ -51,10 +51,14 @@ def toggle_content
     end
 
     def build_toggle_content
-      if toggle.present?
-        h.content_tag(:span, toggle, class: 'alert-toggle')
-      elsif on?
+      return h.content_tag(:span, toggle, class: 'alert-toggle') if toggle.present?
+      if on?
```

Comment:
**not a blocker** (more of a tiny improvement), but you could get rid of the if/else completely by turning this if into:

```ruby
on? alert_on_content : alert_off_content
```
and move the content of those to their associated method names.